src/firmware/app/control/wheel_controller.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include "firmware/shared/circular_buffer.h"
+
+typedef struct WheelController WheelController_t;
+
+/*
+ * NOTE: THIS FUNCTIONS IN THIS FILE OPERATE UNDER THE ASSUMPTION THAT THEY ARE BEING USED
+ * IN A DISCRETE TIME CONTROLLED SYSTEM. THIS MEANS THAT FOR A CONSTANT INTERPOLATION
+ * PERIOD OF 'T' ONLY ONE DATA SAMPLE IS ADDED TO THE SAMPLE BUFFER AND ONLY ONE COMMAND
+ * IS ADDED TO THE COMMAND BUFFER.
+ *
+ * THIS MEANS THAT OVER THE PERIOD OF 'T' SECONDS THERE IS NO CHANGE TO THE APPLIED
+ * VOLTAGE. THIS IS KNOWN AS 'ZERO-ORDER-HOLD' (ZOH).
+ *
+ * For more information:
+ * https://ethz.ch/content/dam/ethz/special-interest/mavt/dynamic-systems-n-control/idsc-dam/Lectures/Digital-Control-Systems/Slides_DigReg_2013.pdf
+ *
+ */
```

Comment:
@garethellis0

Reply:
Ah, I see, could we perhaps give a mock example of how you might use this library? Something with a `delay` call in it?

Reply:
Added